tests/tests/tier0/monitor-system-status-ctrl-stop/test_monitor_system_status_ctrl_stop.py

```diff
@@ -29,14 +31,22 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
     if result != 0:
         raise Exception(f"Failed to stop bluechi-controller: {output}")
 
-    # wait a bit to process all events
-    time.sleep(1)
-
-    result, output = ctrl.exec_run("cat /tmp/events")
-    if result != 0:
-        raise Exception(f"Failed to get events file: {output}")
-
-    assert output == "down,"
+    LOGGER.info("Checking events processed by the system monitor...")
+    expected_events = ["down"]
+    while True:
+        result, output = ctrl.exec_run("cat /tmp/events")
+        if result != 0:
+            raise Exception(f"Unexpected error while getting events file: {output}")
+
+        events = output.split(",")
+        LOGGER.info(f"Got monitored events: '{events}', comparing with expected '{expected_events}'")
```

Comment:
Wouldn't debug log level be more appropriate here?

Reply:
Since the timeout from [conftest.py](https://github.com/eclipse-bluechi/bluechi/blob/main/tests/conftest.py#L17) will cause an error that hints at the event lists here, it'll be useful to view those logs by default - to see what is in the event list.

Reply:
The question is how often we are going to debug, most of the time it will just be increasing the size of integration test artifacts, no?

Reply:
I don't think that adding a few log lines causes a big increase in size - also compared to the host logs. Compared with the potential benefit to make debugging easier, I think its neglectable. If artifact size is really an issue, e.g. because we keep these for a long time, we can also think about adjusting the retention policy.
I changed most of the logs to `debug`, but want have the comparison log (got vs expected events).